### ISSUE TITLE
Release 5 hangs on plugin enable, when "tests" directory does not exist.

### ISSUE THREAD

**Issue by Wcubed** [AlexDarigan/WAT-GDScript#193]
When I add [release 5](https://github.com/CodeDarigan/WAT-GDScript/releases/tag/v5.0.0) of the addon to godot and clilck "enable" in the list of plugins, godot hangs and crashes.

Error log is an infinite list of:
```
Error: call: res://tests/.import/.import/ <snip> /.import/ : 31
At: modules/gdscript/gdscript_functions.cpp:803
```

Fix: create the "res://tests" directory before enabling the plugin.

**Comment by CodeDarigan** [comment]
Missing return on bad dir opening. Should be resolved by cbcc70c21c9e3097a8e8bc4086feeb9c425e0d7c.

I don't want to auto-create dirs for people. It can get annoying sometimes.

**Comment by Wcubed** [comment]
Good point :smiley:, I can see how having directories suddenly appear might not be desirable.
Hmm, I think I meant more "workaround" instead of "fix" :sweat_smile:.
